lib/unleash/context.rb

```diff
@@ -12,6 +12,7 @@ def initialize(params = {})
       self.user_id     = value_for('userId', params)
       self.session_id  = value_for('sessionId', params)
       self.remote_address = value_for('remoteAddress', params)
+      self.current_time = value_for('currentTime', params, Time.now.utc.iso8601.to_s)
```

Comment:
Do we know if we want to use current_time as the time that the context object is created or when the constraint is evaluated?

Reply:
Such a good question! I see that the SDKs do different things here. I was working off the Java implementation which does this but I see the NodeSDK evaluates the current time during constraint evaluation. I think you're right here (if I'm reading between the lines correctly!) - doing it during evaluation is more correct

Reply:
So given that the other SDKs do this, I want to leave this alone. Thoughts?